docs/architecture/modernize-desktop/deploying-modern-desktop-applications.md

```diff
@@ -0,0 +1,332 @@
+requirement is defined and the moment your company delivers the feature to your
+client.
+
+## The modern application lifecycle
+
+Today, developers write and build the code for an app and then pass the
+generated assets to the IT pros who reconfigure the app and repackage it
+typically in an MSI or more recent App-V packaging format, which is deployed
+through different channels and tools. One of the main problems with this
+approach is commonly known as "packaging paralysis". The problem is that this
+cycle repeats every time there is an App update or an OS Update.
+
+We can see the process reflected on this picture:
+
+![Modern IT](media\deploying-modern-desktop-applications\modern-it.png)
+
+Companies need a way to break in this packaging cycle three independent cycles:
+
+* OS Updates
+* Application Updates
+* Customization
+
+![IT cycles](media\deploying-modern-desktop-applications\it-cycles.png)
+
+This means we should be able to update the underlying OS without having to
+repackage our apps or enable customizations from IT without the need to
+repackage the original developer package.
+
+This radical change leads us to the new and modern IT lifecycle depicted in the
+following picture:
+
+![Microsoft IT tools](media\deploying-modern-desktop-applications\microsoft-it-tools.png)
+
+Developers create the app and generate an MSIX package that IT pros can consume
+and configure without the need of repackaging. Along with the MSIX technology,
+Microsoft has created tools to allow IT to customize and configure packages
+without repackaging.
+
+## MSIX: The next generation of deployment
+
+Before MSIX, we had several packaging technologies available like setup wizards,
+MSI, ClickOnce, App-V, and scripting. Each of  technologies has their strengths and
+Microsoft has decided to pick the best of all to build MSIX. Therefore, MSIX is
+built on the foundations of these existing technologies picking the best of
+each:
+
+* App-V =\> Containerization
+* ClickOnce =\> Auto updating
+* MSI =\> Easy to distribute
+
+With MSIX, we get one installer technology with all these features.
+
+![MSIX](media\deploying-modern-desktop-applications\msix.png)
+
+### Benefits of MSIX
+
+#### Never regret installing an App
+
+MSIX provides a predictable, reliable, and safe deployment. The declarative
+method contained in the package manifest lets the OS keep track of every asset
+your application needs. It also provides a true clean uninstall with no side
+effects.
+
+#### Disk space optimization
+
+MSIX is optimized to reduce the footprint and application has on the user's
+machine disk space. It creates a single instance storage of your files, meaning
+that if you have two different packages with the same DLL, it is not installed
+twice. The platform takes care of that because it knows all the files that are
+installed by a particular app thanks to its declarative nature. It also allows
+you to have different versions of a DLL working side by side.
```

Comment:
Please review my suggestion carefully.

```suggestion
MSIX is optimized to reduce the footprint that an application has on the user's
machine disk space. It creates a single instance storage of your files. That is, 
if you have two different packages with the same DLL, the DLL isn't installed
twice. The platform takes care of that problem because it knows all the files that
a particular app installed thanks to its declarative nature. It also allows
you to have different versions of a DLL working side by side.
```